sdks/typescript/biome.json

```
@@ -0,0 +1,55 @@
+{
+  "$schema": "https://biomejs.dev/schemas/2.0.6/schema.json",
+  "vcs": {
+    "enabled": true,
+    "clientKind": "git",
+    "useIgnoreFile": true
+  },
+  "files": {
+    "includes": [
```

Biome (1.9.4)

[error] Found an unknown key `includes`.

Known keys:


(deserialize)